doc/man1/flux-start.rst

```diff
@@ -15,15 +15,107 @@ DESCRIPTION
 
 .. program:: flux start
 
-:program:`flux start` launches a new Flux instance. By default,
-:program:`flux start` execs a single :man1:`flux-broker` directly, which
-will attempt to use PMI to fetch job information and bootstrap a flux instance.
+:program:`flux start` assists with launching a new Flux instance, which
+consists of one or more :man1:`flux-broker` processes functioning as a
+distributed system.  It is primarily useful in environments that don't run
+Flux natively, or when a standalone Flux instance is required for test,
+development, or post-mortem debugging of another Flux instance.
+
+On systems already running Flux, single-user Flux instances can be more
+conveniently started with :man1:`flux-batch` and :man1:`flux-alloc`.
+The `Flux Administration Guide
+<https://flux-framework.readthedocs.io/en/latest/guides/admin-guide.html>`_
+covers setting up a multi-user Flux "system instance", where Flux natively
+manages a cluster's resources and those commands work ab initio for its users.
```

Comment:
Just a thought: perhaps it would be simpler to say something like "When already running under Flux, single-user Flux instances can be more conveniently started with..." As it is now, some readers may assume a system instance is required to use `flux alloc` and `flux batch`.

However, the reference to the Admin Guide may still be useful so my comment may be off.

Reply:
Great point.  I think I can reword the one sentence and leave the admin guide reference.  Just pushed that - see if it reads ok for you.

Reply:
Yeah, looks good to me!